utils/version.py

```diff
@@ -14,7 +14,7 @@
     {{
         static const char* const DEFINED_VERSION = "{game_version}";
         static const char* const UNKNOWN = "v0.0.0-unknown";
-        static const char* const VERSION = (DEFINED_VERSION[0] == "0") ? UNKNOWN : DEFINED_VERSION;
+        static const char* const VERSION = (DEFINED_VERSION[0] == "\0") ? UNKNOWN : DEFINED_VERSION;
```

Comment:
The `'` thing is important - `"` and `'` mean different things in C++ unlike Python, so it should be `'\0'`.
`"\0"` is a c-string, and `'\0'` is a character.